src/parser/Parser.spec.ts

```diff
@@ -1124,6 +1127,71 @@ describe('parser', () => {
             expect(type.returnType).to.be.instanceof(CustomType);
         });
     });
+
+    describe('symbolTable', () => {
```

Comment:
Can you add tests to ensure `for` and `for-each` loop variable are included in the symbol table? 

`for i = 0 to 10 step 1` and `for each thing in items`